Cache/Main.lean

```diff
@@ -44,12 +44,16 @@ Which will download the cache for:
 * Every Lean file inside 'Mathlib/Data/'
 * Everything that's needed for the above"
 
-open System (FilePath) in
+open Lean System in
 /-- Note that this normalizes the path strings, which is needed when running from a unix shell
 (which uses `/` in paths) on windows (which uses `\` in paths) as otherwise our filename keys won't
 match. -/
-def toPaths (args : List String) : List FilePath :=
-  args.map (FilePath.mk · |>.normalize)
+def toPaths (args : List String) : IO (List FilePath) :=
+  args.mapM fun arg =>
+    if arg.endsWith ".lean" then
+      return FilePath.mk arg |>.normalize
+    else do
+      return mkFilePath (arg.toName.components.map Name.toString) |>.withExtension "lean"
```

Comment:
why is this in IO?

Reply:
Oops. It had been when I was using some Lean internal function that needed to be in IO, then never cleaned up.